### PR TITLE
release: v3.6.0 - official feature config support

### PR DESCRIPTION
**Pull request by AhyoungRyu**
https://sendbird.atlassian.net/browse/SDKRLSD-886

### Feat:
* Official support for Feature Configuration
  - You can now configure the features of UIKit through the `uikitOptions` prop of `<SendbirdProvider />` or `<App />` component. You can also find the detailed sample usage from [SAMPLE.md#UIKit-Configuration-Samples](./SAMPLES.md#UIKit-Configuration-Samples)
  - The minimum `@sendbird/chat` version has been increased to 4.9.2.
```jsx
  <SendbirdUIKitContainer
    uikitOptions={{
      common: {
        enableUsingDefaultUserProfile: true,
      },
      groupChannel: {
        enableMention: false,
        enableOgtag: true,
        enableReaction: true,
        enableTypingIndicator: true,
        input: {
          camera: {
            enablePhoto: true,
            enableVideo: true,
          },
          gallery: {
            enablePhoto: true,
            enableVideo: true,
          },
          enableDocument: true,
        },
      },
      groupChannelList: {
        enableTypingIndicator: true,
        enableMessageReceiptStatus: true,
      },
      groupChannelSettings: {
        enableMessageSearch: true,
      },
      openChannel: {
        enableOgtag: true,
        input: {
          camera: {
            enablePhoto: true,
            enableVideo: true,
          },
          gallery: {
            enablePhoto: true,
            enableVideo: true,
          },
          enableDocument: true,
        },
      },
    }}
  />
```

FYI, no new features or fixes have been added since we rolled out v.3.5.2 https://github.com/sendbird/sendbird-uikit-react/pull/656

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,57 @@
 # Changelog - v3
+## [v3.6.0] (June 28 2023)
+
+### Feat:
+* Official support for Feature Configuration
+  - You can now configure the features of UIKit through the `uikitOptions` prop of `<SendbirdProvider />` or `<App />` component. You can also find the detailed sample usage from [SAMPLE.md#UIKit-Configuration-Samples](./SAMPLES.md#UIKit-Configuration-Samples)
+  - The minimum `@sendbird/chat` version has been increased to 4.9.2.
+```jsx
+  <SendbirdUIKitContainer
```
Comment (HoonBaek) [inline CHANGELOG.md:9]: Does this mean SendbirdProvider ?

Comment (AhyoungRyu) [reply CHANGELOG.md]: Oh right. Thanks for catching it out. https://github.com/sendbird/sendbird-uikit-react/pull/662/commits/a5cb0184590f0f2075e41d4c3b3c3977f1dc3c95 Fixed in here.

**Review by HoonBaek — APPROVED** [review]
LGTM

nit: it should be small `b` in the SendbirdProvider

**Comment by roderickhsiao** [comment]
it seems like this version introduced `@sendbird/uikit-tools` are not complied to cjs kind of a breaking changes if user are not including the package in their transpile seeting.

In our app example (NextJS), adding 
```
    transpilePackages: ['@sendbird/uikit-tools'],
```

will fix it, but not sure if we should fix the issue upstream that publishing the package should compile or marked as module package itself.

**Comment by sravan-s** [comment]
@roderickhsiao  can you tell us more about your nextjs setup -> ie, versions etc

**Comment by roderickhsiao** [comment]
For sure,

I think it's not particularly related to Nextjs, its more a commonjs issue

next@13.4.7, (pages folder instead of app)

and we are using directly core component like

```js
import SBConversation from '@sendbird/uikit-react/Channel';
import SBChannelSettings from '@sendbird/uikit-react/ChannelSettings';
import SBProvider from '@sendbird/uikit-react/SendbirdProvider';
```

Screenshot 2023-06-28 at 7 39 42 PM

https://www.npmjs.com/package/@sendbird/uikit-tools?activeTab=code

both main/module are pointing to the same file in package.json which the index module is es-module rather than comonjs

```
  "main": "./dist/index.js",
  "module": "./dist/index.js",
```

**Comment by sravan-s** [comment]
YEah, so I was curious because we tested in nextjs and it was working fine.. But turns out it was on app-directory.. It indeed fails with `pages`. We are discussing and will share a fix soon :)

**Comment by roderickhsiao** [comment]
Thanks!

Not a blocker as there is workaround but many thanks for prompt response
and debug!

**Comment by sravan-s** [comment]
@bang9  Updated @sendbird/uikit-tools ~ should be okay from now on

**Comment by roderickhsiao** [comment]
verified `0.0.1-alpha.39` fixed the issue :) many thanks!

**Pull request merged**